core/consensus/src/synchronization.rs

```diff
@@ -263,11 +242,7 @@ impl<Adapter: SynchronizationAdapter> OverlordSynchronization<Adapter> {
                 .verify_txs_sync(
                     ctx.clone(),
                     consenting_height,
-                    consenting_rich_block
```

Comment:
Enough transactions have been requested so that there is no need to request them again.